### PR TITLE
fixed the incorrect error message in value addition

### PR DESCRIPTION
**Pull request by G-Joe**
This change resolves the #345  by rewriting the error message 

## Summary by CodeRabbit

- **Bug Fixes**
  - Corrected error messages in ProgressIndicator component for better clarity.
  
- **Refactor**
  - Improved code readability and maintainability in the ProgressIndicator component.

- **Chores**
  - Modified VSCode settings to adjust ESLint actions on save from `true` to `explicit`.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "typescriptreact"
   ],
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```
Comment (kotAPI) [inline .vscode/settings.json:10]: why do we need this? wont this disable linting on save?

Comment (G-Joe) [reply .vscode/settings.json:10]: it is supposed to have one value  "source.fixAll.eslint": true 

Do I fix and recreate a pull request?

Comment (G-Joe) [reply .vscode/settings.json:10]: TO disbale linting on save, I would have to set 

`  "source.fixAll.eslint": false `
  which is not the case here

**Review by kotAPI — APPROVED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (kotAPI) [reply .vscode/settings.json]: but we want to lint on save,
did it cause any issue when you did that? just trying to understand if it had caused you any problem for you to push that change.

Comment (G-Joe) [reply .vscode/settings.json]: Oh no it didn't. Only thing that happened was I upgraded my VS code to a newer version, probably that is where it was added from

**Pull request merged**